docs/source/modules/tutorial.rst

```diff
@@ -0,0 +1,653 @@
+   file that you just successfully ran is an instance of this template, and this template
+   declares how models must be structured to be scored. For now, we will just submit the
+   AlexNet model as is.
+
+
+
+
+Submitting a Model to Brain-Score.org Part 2: Upload
+====================================================
+
+If you made it this far, you are ready to upload your AlexNet model
+and get a Brain-Score! In a nutshell, all this step ensues is zipping
+the folder and making sure the files to submit are in the right place.
+
+1. Right now, the working code we have confirmed is ready to be submitted is
+   in the ``pytorch.py`` file. This file is mainly an example file, and
+   thus we do not really want to submit it - instead, we are going to
+   make a copy of it, rename it, and submit *that* version.
+2. Before we do this, it is best to go ahead and make a folder in the
+   root ``brainscore-brief`` directory to house all your submissions.
+   This way, you can have a nice place to keep your submissions and
+   reference them later if need be. For example, I made one called
+   ``my_model_submissions`` located inside the project root (``brainscore-brief``),
+   as seen below:
+
+    .. image:: /docs/source/modules/tutorial_screenshots/mms.png
+      :width: 600
+
+3. We are now going to make the sub-folders necessary for submission.
+   In general, the submission package will be a zip folder with a few things in
+   it. It is important to get the folder “levels" right, or the website will not
+   be able to parse the submission package and start running the correct code.
+   So, this step in the guide is just about building this submission package.
+   See below for the breakdown of zip file we will submit, with the various levels of the folders. Note the
+   *two* ``__init__.py`` files in both the ``models`` folder and root: ::
+
+
+    my_alexnet_submission (main folder)
+        models (subfolder)
+            base_models.py
+            __init__.py
+        __init__.py
+        setup.py
+
+4. Now we will start making the submission package. In your ``my_model_submissions`` folder,
+   create a new folder (that we will eventually zip to submit) called ``my_alexnet_submission``.
+   In that newly created folder, create (yet another) folder called ``models``.
+   You can see we are building the package up as explained above.
+   Your IDE file/project explorer should look something like this below at this point:
+    .. image:: /docs/source/modules/tutorial_screenshots/subfolders.png
+      :width: 600
+
+5. Next, we are going to add the ``setup.py`` file into the ``my_alexnet_submission`` folder.
+   There are a few ways to do this, but the easiest is just to navigate into the
+   ``sample_model_submission`` folder, and you will see a ``setup.py`` file there. We are going
+   to copy that and place it inside of the ``my_alexnet_submission`` folder. You should
```

Comment:
as above, instead of copying everything over I think it is easier for users to just modify the local `sample-model-submission`

Reply:
(same ping as above)